openmetadata-ui/src/main/resources/ui/playwright/e2e/Features/DataQuality/AddTestCaseNewFlow.spec.ts

```diff
@@ -103,8 +103,6 @@ test.describe('Add TestCase New Flow', () => {
       expect(response.status()).toBe(201);
       expect(ingestionPipelineCalled).toBe(false);
     }
-
-    await toastNotification(page, 'Test case created successfully.');
```

Comment:
Why we removed the asset?

Reply:
We don't need to validate the notification, as I am validating API success.